src/spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -82,7 +82,9 @@ def __init__(
             all_annotations=all_annotations,
             **neo_kwargs,
         )
-        self._kwargs.update(dict(folder_path=str(Path(folder_path).absolute())))
+        self._kwargs.update(
+            dict(folder_path=str(Path(folder_path).absolute())), ignore_timestamps_errors=ignore_timestamps_errors
```

Comment:
This option is very very dangerous because it do not ensure channel alignement!

Reply:
unrelated though. If it's an argument, it should be in the `self._kwargs`. The misalignment problem is NEO-related